### DISCUSSION TITLE
Installation Script for Linux

### DISCUSSION THREAD

**Discussion by technoslothing** [OliveTin/OliveTin#77] [Ideas]
I am not sure if this was already brought up but I made a simple installation script for the application on linux.  If it would be of interests to others.

https://github.com/Holyghosts/olivetin-installer

**Comment by jamesread** [comment]
Hey @Holyghosts , this is nice! Thanks for taking the time to do that. 

I'm just about to make a new release that switches the URL for downloading OliveTin - from the version to "latest", see #68. This would break your script, so I'll let you know here when this change happens. 

Would you be interested in submitting your script as a pull request into the project officially? If you fork this project, create a feature branch and add your script into /var/installscript/, and add it in . goreleaser.yml under archives -> files, then submit a pull request - I'd accept that :-) 

I'd credit you as the author in the release notes, etc :-)

**Reply by technoslothing** [reply]
Let me rework it a bit to accept changes to the URL and I can add it to the fork :)

**Reply by jamesread** [reply]
@Holyghosts This URL should now always work, it no longer needs the version number; https://github.com/OliveTin/OliveTin/releases/latest/download/OliveTin-linux-amd64.tar.gz

**Reply by technoslothing** [reply]
Sorry for the delay, I have reworked the script to be a bit more robust.  I tested a few times on pop-os but it may need additional testing just to be safe.  I also added the ability to add custom config and download url.  If not provided it will use the defaults. I have pushed it to the new fork.

**Reply by technoslothing** [reply]
Also, I want to add some improvements when time permits to check if OliveTin is already installed and some regex validation for the parameters.
